compiler/repl-service/server/src/main/scala/com/digitalasset/daml/lf/ReplServiceMain.scala

```diff
@@ -202,18 +224,22 @@ class ReplService(val clients: Participants[LedgerClient], ec: ExecutionContext,
       case (pkgId, pkg) => pkg.modules.contains(DottedName.assertFromString("Daml.Script"))
     }.get
 
+    // Load the new package. All dependencies should already be loaded.
+    compiledPackages.addPackage(linePkgId, pkg) match {
+      case ResultDone(()) => ()
+      case ResultNeedPackage(pkgId, resume) =>
+        throw new RuntimeException(s"Missing package $pkgId")
```

Comment:
Are exceptions in grpc handlers automagically caught and turned into error responses or do we need to call `respObs.onError`?

Reply:
They seem to be, the error is displayed in the REPL console and the `Left` branch on `scriptRes` is taken in `handleLine`.